docs/section-7/feedforward-control.md

```diff
@@ -0,0 +1,213 @@
+```java
+// assume kS, kV, and kA are defined
+double vel = 5;
+double accel = 1;
+double feedforwardVolts = kS * Math.signum(vel) + kV * vel + kA * accel;
+motor.setVoltage(feedforwardVolts);
+```
+!!! warning
+    The amount of voltage calculated is the amount of voltage used to **MAINTAIN** the motor at the specified velocity and acceleration. When `motor.setVoltage(feedforwardVolts)` is run, that does not automatically drive the motor to the specified velocity and acceleration. If the code was run when the motor is at rest, then the voltage will be used the overcome the static friction and accelerate the motor, not to maintain the motor at the specified velocity and acceleration as the motor is not at the specified velocity.
+
+!!! warning
+    The code excerpt is only meant to show how feedforward works. This is not how we actually implement feedforward, but should give you a better idea of the inner workings of feedforward.
+
+In addition, feedforward can also be used for elevators and arms. There is one additional constant \\(k_{g}\\) which is used to counteract the force of gravity.
+
+## Tuning and System Idenfication
+Similar to PID, you can tune values by manually guessing and checking.
+
+[Click here to try tuning a feedforward controller. SCROLL TO "PURE FEEDFORWARD CONTROL", SKIP EVERYTHING ELSE](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/introduction/tuning-flywheel.html)
+
+Follow the instructions and see if you can get the optimal tuning solution. The model simulates a flywheel shooter mechanism and halfway through the simulation it shoots a ball. **DO NOT SKIP THIS PRACTICE**
+
+While manual tuning works, WPILIB provides a way to generate kS, kV, and kA, called System Identification, or SysID for short.
+
+!!! warning
+    Do not move on if you don't know how [lambdas/consumers](https://docs.wpilib.org/en/stable/docs/software/basic-programming/functions-as-data.html) work and the [Java Unit library](https://docs.wpilib.org/en/stable/docs/software/basic-programming/java-units.html).
+
+System Identification is the process of determining a mathematical model for the behavior of a system through statistical anaylsis of its inputs and outputs. SysID has a process to determine kS, kV, and kA for the motor, so you don't have to do any tuning! They also provide PID values, but treat them as a "starting point" for further tuning.
+
+Read the following WPILIB articles:
+
+- [Creating an Identification Routine](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/system-identification/creating-routine.html)
+- [Running the Identification Routine](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/system-identification/running-routine.html)
+
+<hr>
+
+For those that want a quick summary:
+
+The SysID tool runs two types of tests: 
+
+- Quasistatic: mechanism is gradually sped up (determines amount of voltage needed for velocity)
```

Comment:
please use some sort of formatting to separate these four-element lines
- "text" text (text) is really hard for the eye to understand how to read

Reply:
done